assets/src/logging.js

```diff
@@ -1,3 +1,4 @@
+'use strict';
```

Comment:
If everything is already linted as strict, why do we need to still add this header?